tests/commands/build/test_call.py

```diff
@@ -11,11 +11,11 @@ def test_specific_app(build_command, first_app, second_app):
         "second": second_app,
     }
 
-    # Configure no command line options
-    options, _ = build_command.parse_options([])
+    # Configure command line options for building a specific app
+    options, _ = build_command.parse_options(["--app", "first"])
```

Comment:
The existing test is verifying something specific. If you're adding behavior, you need to add a *new* test for the new test configuration, not just modify an existing one.